Lesson_11/l11_h01_generators.py

```diff
@@ -0,0 +1,34 @@
+"""
+HW:11[Volkov Anton].
+
+Write a generator function that takes a range value and chunk_size as input
+and produces output as lists with length equal to the chunk size.
+Example:
+Call: chunk_generator(range(11), chunk_size=3)
+Output:
+[0, 1, 2]
+[3, 4, 5]
+[6, 7, 8]
+[9, 10]
+"""
+
+
+def chunk_generator(range_value, chunk_size):
+    """Iterate over range value and store i in list of length chunk_size."""
+    part = []
```

Comment:
A little bit complicated - two yields, two places with part = []. Such code for sure could be optimized.
Pay attention that usually generators use while cycles. While cycle could simplify your life )
Possible solution with while:
```
def chunks_generator(range_obj, chunk_size=3):
    target_list = list(range_obj)
    while target_list:
        yield target_list[:chunk_size]
        target_list = target_list[chunk_size:]
```